src/main/shadow/cljs/ui/views/runtimes.cljs

```diff
@@ -0,0 +1,141 @@
+      [:runtime-id
+       :runtime-info
+       :supported-ops]))
+
+  (render
+    (let [{:keys [lang build-id host type since user-agent desc]} runtime-info]
+
+      (<< [:tr.align-top
+           [:td.pl-2.text-right runtime-id]
+           [:td.pl-2.whitespace-no-wrap (if-not lang "-" (name lang))]
+           [:td.pl-2.whitespace-no-wrap
+            (when build-id
+              (<< [:a {:href (str "/build/" (name build-id))} (name build-id)]))]
+           [:td.pl-2.whitespace-no-wrap (when host (name host))]
+           [:td.pl-2.whitespace-no-wrap (age-display since)]
+           [:td.pl-2.truncate (or desc user-agent "")]]
+        [:tr
+         [:td.border-b.py-2]
+         [:td.border-b.py-2 {:colSpan 5}
+          (when (contains? supported-ops :cljs-eval)
+            (<< [:a
+                 {:class inspect/css-button
+                  :href (str "/runtime/" runtime-id "/cljs-eval")}
+                 "cljs eval"]))
+          (when (contains? supported-ops :clj-eval)
+            (<< [:a
+                 {:class inspect/css-button
+                  :href (str "/runtime/" runtime-id "/repl")}
+                 "clj eval"]))
+          (when (contains? supported-ops :db/get-databases)
+            (<< [:a
+                 {:class inspect/css-button
+                  :href (str "/runtime/" runtime-id "/db-explorer")}
+                 "db explorer"]))]])
+      (<< [:tr.bg-white
+           [:td.max-w-0.w-full.px-6.py-4.whitespace-nowrap.text-sm.text-gray-900
+            [:div.flex
+             [:a.inline-flex.space-x-2.pl-1.group.truncate.text-sm {:href (if build-id
+                                                                            (str "/build/" (name build-id))
+                                                                            "#")}
+              [:div
+               (build-lang lang)]
+              [:div.inline-flex.space-x-2.pl-1
+               [:p.text-gray-500.truncate.group-hover:text-gray-900 runtime-id]
+               [:p.text-gray-500.truncate.group-hover:text-gray-900 (when build-id (name build-id))]]]]]
+           [:td.px-6.py-4.text-right.whitespace-nowrap.text-sm.text-gray-500
+            [:p.text-gray-900.font-medium (when host (name host))] (or desc user-agent "")]
+           [:td.px-6.py-4.text-right.whitespace-nowrap.text-sm.text-gray-500 (age-display since)]]))))
+
+(defn ui-runtime-listing [runtimes]
+  (<< [:table.min-w-full.divide-y.divide-gray-200
+       [:thead
+        [:tr
+         [:th.px-6.py-3.bg-gray-50.text-left.text-xs.font-medium.text-gray-500.uppercase.tracking-wider "build"]
+         [:th.px-6.py-3.bg-gray-50.text-right.text-xs.font-medium.text-gray-500.uppercase.tracking-wider "host"]
+         [:th.px-6.py-3.bg-gray-50.text-right.text-xs.font-medium.text-gray-500.uppercase.tracking-wider "since"]]]
+       [:tbody.bg-white.divide-y.divide-gray-200
+        (sg/render-seq runtimes identity ui-runtime-overview)]]))
+
+(defc ui-runtime-overview-mobile
```

Comment:
`mobile` is a bit misleading since I do not expect this to ever be openend on an actual `mobile`. Maybe better to call it `narrow`.